srtcore/buffer_rcv.cpp

```diff
@@ -1007,9 +1007,10 @@ void CRcvBufferNew::updateTsbPdTimeBase(uint32_t usPktTimestamp)
 string CRcvBufferNew::strFullnessState(int iFirstUnackSeqNo, const time_point& tsNow) const
 {
     stringstream ss;
+    ss << "iFirstUnackSeqNo=" << iFirstUnackSeqNo << " m_iStartSeqNo=" << m_iStartSeqNo
+       << " m_iStartPos=" << m_iStartPos << " m_iMaxPosInc=" << m_iMaxPosInc << ". ";
```

Comment:
This function is only used for error logs, so I think it's fine to add more info.
In production env without heavy log, these bugs are very hard to reproduce, so we need more information in the error log.

Reply:
I found another unsolved "No room" issue in my production env, but I can't reproduce it, so it's good to add more info into the no room log.

Reply:
There is always some debate around increasing logs to help investigate issues and decreasing them to reduce the burden of storing those logs on premises where you pay per byte.
I totally agree this additional information could help investigate some issues.
My concern (and I also get some pressure from product teams on that) is that it may be called quite often with the "No room to store incoming packet..." log message. The message is already quite long, and there is even some sentiment about reducing it somehow.

Maybe we should think about enabling it on demand. E.g. depending on loglevel. If "debug" level is specified, include it in the error message. Or maybe shorten it somehow. 🤔

Reply:
> depending on loglevel. If "debug" level is specified, include it in the error message.

Applied.

Reply:
🤝